coupons_test.go

```diff
@@ -17,117 +17,87 @@ import (
 // Because Recurly supports partial updates, it's important that only defined
 // fields are handled properly -- including types like booleans and integers which
 // have zero values that we want to send.
-func TestCoupons_Encoding(t *testing.T) {
+func TestCoupons_EncodingDecoding(t *testing.T) {
```

Comment:
This can just be called `TestCoupons_Encoding`, that's what these tests are labeled throughout the project.